packages/codemirror-blocks/src/edits/commitChanges.ts

```diff
@@ -41,45 +39,40 @@ export const commitChanges =
     focusHint?: FocusHint | -1,
     astHint?: AST,
     annt?: string | false
-  ): AppThunk<Result<{ newAST: AST; focusId?: string }>> =>
+  ): AppThunk<{ newAST: AST; focusId?: string }> =>
   (dispatch, getState) => {
-    try {
```

Comment:
It took a moment for me to convince myself that this is no longer exception-generating code. But OK, I'm convinced!